source/includes/reference/_users.md

```diff
@@ -62,7 +62,67 @@ details.address.city        | Address city name          | Text
 details.address.state  | Address state code State code. Required if country is US, CA, AU, BR. | Text
 details.address.occupation  | User occupation. Required for US, CA, JP         | Text
 
+## Get the currently logged in user
 
+> Example Request:
+
+```shell
+curl -X GET https://api.sandbox.transferwise.tech/v1/me \
+     -H "Authorization: Bearer <your api token>" 
+```
+
+> Example Response:
+
+```json
+{
+  "id": 294205,
+  "name": "Oliver Wilson",
+  "email": "oliver@gmail.com",
+  "active": true,
+  "details": {
+    "firstName": "Oliver",
+    "lastName": "Wilson",
+    "phoneNumber": "+3725064992",
+    "occupation": "",
+    "address": {
+      "city": "Tallinn",
+      "countryCode": "EE",
+      "postCode": "12112",
+      "state": "",
+      "firstLine": "Salu tee 134"
+    },
+    "dateOfBirth": "1977-07-01",
+    "avatar": "https://lh6.googleusercontent.com/-XzeFZ2PJE1A/AAAAAAAAAAI/AAAAAAAAAAA/RvuvhXFsqs0/photo.jpg",
+    "primaryAddress": 236532
+  }
+}
+```
+Get authenticated user details for the user's token submitted in the `Authorization` header. Response includes also personal user profile info.
+
+### Request
+**`GET https://api.sandbox.transferwise.tech/v1/me`**
+
+
+### Response
+Field                   | Description                    | Format
+---------               | -------                        | -----------
+id                      | userId                         | Integer
+name                    | User full name                 | Text
+email                   | User email                     | Text
+active                  | If user is active or not       | Boolean
+details.firstName       | User first name                | Text
+details.lastName        | User lastname                  | Text
+details.phoneNumber     | Phone number                   | Text
+details.dateOfBirth     | Date of birth                  | YYYY-MM-DD
+details.occupation      | Person occupation              | Text
+details.avatar          | Link to person avatar image    | Text
+details.primaryAddress  | Address object id              | Integer
+details.address.country  | Address country code in 2 digits. "US" for example            | Text
```

Comment:
Should be `countryCode` as per example response above. Also are we sure this is 2 characters? In many places we use 3 character ISO code.

Reply:
Thanks, yes it is 2 char